products/BellHybrid/apps/application-bell-onboarding/windows/OnBoardingOnOffWindow.cpp

```diff
@@ -28,7 +28,7 @@ namespace style::onboarding
     namespace text
     {
         constexpr auto marginLeft   = 15;
-        constexpr auto minimumSizeX = 81;
+        constexpr auto minimumSizeX = 100;
```

Comment:
Why the change in size?

Reply:
I changed that so community fonts would fit. Reverting since we dont wanna make sure they do